src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -60,6 +66,11 @@ public class RoleManager<TRole> : IDisposable where TRole : class
         protected IRoleStore<TRole> Store { get; private set; }
 
         /// <summary>
+        ///     Used to create scopes and messages for logging
+        /// </summary>
+        protected internal virtual IdentityLogger IdentityLogger { get; set; }
```

Comment:
Yeah the scope method calls it in each manager class  while it is used in a lot places in the test code. If I change the Logger property to `BaseLogger` then the call would be `Logger.BaseLogger` or if we are fine that the test code and the method have `Logger.Logger` then i can make the change. Personally I don't like the `Logger.Logger` call